pkg/mutate/squashfs.go

```diff
@@ -20,9 +20,10 @@ import (
 const layerMediaType types.MediaType = "application/vnd.sylabs.image.layer.v1.squashfs"
 
 type squashfsConverter struct {
-	converter string   // Path to converter program.
-	args      []string // Arguments required for converter program.
-	dir       string   // Working directory.
+	converter         string   // Path to converter program.
+	args              []string // Arguments required for converter program.
+	dir               string   // Working directory.
+	noConvertWhiteout bool     // Skip default conversion of whiteout markers from AUFS -> OverlayFS
 }
```

Comment:
Nit: use the positive version here (to avoid `!noConvertWhiteout` type of situation.)

```suggestion
	convertWhiteout bool     // Convert whiteout markers from AUFS -> OverlayFS.
```